sdk/core/core-client/src/authorizeRequestOnTenantChallenge.ts

```diff
@@ -25,6 +25,12 @@ const Constants = {
   },
 };
 
+function isUuid(text: string): boolean {
+  return /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/.test(
```

Comment:
FYI this is the regex used in the uuid package: https://github.com/uuidjs/uuid/blob/main/src/regex.js

Reply:
ah right, since v4 uuids are constrained further and can't just use any valid digits in certain positions.